zephyr/Kconfig

```diff
@@ -0,0 +1,197 @@
+menuconfig OPEN62541
```

Comment:
We try to keep the top-level directory clean.
Please move this to /arch/zephyr.
Also the eventloop implementation should go to /arch/zephyr.
We will then move the existing eventloop to /arch/posix.

Reply:
Zephyr requires the two files `Kconfig` and `module.yml` at this specific location to make the library an zephyr module: https://docs.zephyrproject.org/latest/develop/modules.html

Reply:
Moving `arch/eventloop_zephyr` to `arch/zephyr` is done.